collect_app/src/main/java/org/odk/collect/android/utilities/ImageConverter.java

```diff
@@ -39,9 +39,26 @@
     private ImageConverter() {
     }
 
+    /**
+     * Before proceed with scaling or rotating, make sure existing exif information is store/restored.
+     * @author Khuong Ninh (khuong.ninh@it-development.com)
+     */
     public static void execute(String imagePath, QuestionWidget questionWidget, Context context) {
+        ExifInterface exif = null;
```

Comment:
Could this maybe align with how you named it in the comment above (store/restore)? I mean this could be in a separate method called `storeExifAttributes()` and the code below also could be in a separate method called `restoreExifAttributes()` then the code would be cleaner.

Reply:
For simplicity's sake, I would avoid extra unnecessary methods/functions' overhead which (in this case) only fetch exif via constructor and then save it back via built-in function (saveAttributes).

Reply:
generally i like more my approach but I'm not going to insist let's keep it as is.